### PR TITLE
docs: vault-k8s and cert-manager

### PR DESCRIPTION
**Pull request by tvoran**
Adding an example of configuring vault-k8s to use cert-manager for managing the webhook certs. This allows multiple replicas of the injector to be deployed, without having to rely on a leader-elector service.

### CODE REVIEW

**Review by calvn — APPROVED** [review]
Very neat!

File: website/content/docs/platform/k8s/helm/examples/injector-tls-cert-manager.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+```shell
+$ kubectl -n vault apply -f injector-certificate.yaml
+certificate.cert-manager.io/injector-certificate created
+
+$ kubectl -n vault get certificates injector-certificate -o wide
+NAME                   READY   SECRET         ISSUER               STATUS                                          AGE
+injector-certificate   True    injector-tls   injector-ca-issuer   Certificate is up to date and has not expired   41s
+
+$ kubectl -n vault get secret injector-tls
+kubectl -n vault get secret injector-tls
```
Comment (calvn) [inline website/content/docs/platform/k8s/helm/examples/injector-tls-cert-manager.mdx:125]: Duplicate line here
```suggestion
```

File: website/content/docs/platform/k8s/helm/examples/injector-tls-cert-manager.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+$ helm repo update
+$ helm install cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --set installCRDs=true
+```
+
+## Create a Certificate Authority (CA)
+
+For this example we will bootstrap a self-signed certificate authority (CA) [Issuer](https://cert-manager.io/docs/configuration/). If you already have a ClusterIssuer configured for your cluster, you may skip this step.
```
Comment (benashz) [inline website/content/docs/platform/k8s/helm/examples/injector-tls-cert-manager.mdx:28]:
```suggestion
For this example we will bootstrap a self-signed certificate authority (CA) [Issuer](https://cert-manager.io/docs/configuration/). If you already have a [ClusterIssuer](https://cert-manager.io/docs/concepts/issuer/) configured for your cluster, you may skip this step.
```

**Review by benashz — APPROVED** [review]
Looks great!

**Pull request merged**